src/entities/applet/model/hooks/useInProgressEntities.ts

```diff
@@ -49,12 +55,16 @@ function useInProgressEntities(appletId: string) {
     return new Promise(resolve => {
       const event = selectEntityEvent(flowId, eventId);
 
-      if (event) {
+      if (event && !event.endAt) {
```

Comment:
Can we use functions like event.isInProgress() or event.isCompleted?

Reply:
No, unfortunatelly